app-router/installation.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Installation
+---
+
+import CatchAllExample from "/snippets/app-router/catch-all.mdx"
+import ProvidersExample from "/snippets/app-router/providers.mdx"
+import LayoutExample from "/snippets/app-router/layout.mdx"
+import ApiHandlerExample from "/snippets/app-router/api-handler.mdx"
+import RuntimeExample from "/snippets/reference/runtime.mdx"
+import ClientExample from "/snippets/reference/client.mdx"
+import NextConfigExample from "/snippets/reference/next-config.mdx"
+
+The fastest way to get started with Makeswift on a new Next.js project is to follow the
+[quickstart](/quickstart) guide. If you have an existing Next.js application
```

Comment:
Since the quickstart steers users towards `npx makeswift@latest init`, we should update the `basic-typescript` example to use App Router as soon as we ship `0.15.0` and before we release the docs so the CLI uses App Router by default.